src/patternfly/components/Banner/banner.scss

```diff
@@ -57,4 +64,37 @@
     z-index: var(--pf-c-banner--m-sticky--ZIndex);
     box-shadow: var(--pf-c-banner--m-sticky--BoxShadow);
   }
+
+  a {
+    font-weight: var(--pf-c-banner--link--FontWeight);
+    color: var(--pf-c-banner--link--Color);
+
+    &:hover {
+      color: var(--pf-c-banner--link--hover--Color);
+    }
+
+    // stylelint-disable selector-no-qualifying-type
+    &.pf-m-disabled {
+      color: var(--pf-c-banner--link--disabled--Color);
+      text-decoration: none;
+      pointer-events: auto;
+      cursor: not-allowed;
+    }
+    // stylelint-enable selector-no-qualifying-type
+  }
+
+  .pf-c-button.pf-m-link {
```

Comment:
targeting `.pf-m-inline` is probably more ideal here since that's what we want to style. I suspect we want to leave the plain link button variant alone, as we would the other button variants like primary, secondary, tertiary, etc. my previous comment was just about being able to target the inline button links using link vars instead of always being tied to the banner color, which you've done here.